src/plugins/vis_type_table/public/components/table_vis_controls.tsx

```diff
@@ -85,6 +93,20 @@ export const TableVisControls = memo(
       </EuiButtonEmpty>
     );
 
+    const downloadButton = detectedFormulasInTable ? (
+      <EuiToolTip
+        position="top"
+        content={i18n.translate('visTypeTable.vis.controls.exportButtonFormulasWarning', {
+          defaultMessage:
+            'Your CSV contains characters which spreadsheet application can interpret as formulas',
```

Comment:
Looks like the same typo in all the text you've added:

```suggestion
            'Your CSV contains characters which spreadsheet applications can interpret as formulas',
```